src/main/scala/rps/GameRepository.scala

```diff
@@ -8,23 +8,25 @@ import scala.collection.concurrent.TrieMap
 trait GameRepository {
 
   def saveGameOutcome(
+      theId: String,
```

Comment:
👍 super minor thing. The `theVariable` naming is a bit uncommon (at least in Scala at buildo) We usually just use `id` or `gameId` for these cases ;)

Reply:
Gotcha! I will type it accordingly in the next project!